src/server/models/db/schema.prisma

```diff
@@ -130,14 +130,7 @@ model Feedback {
 // actions that can be performed on anything that is not a ListItem. ListItems have their own events.
 enum AuditEvent {
   NEW
-  OUT_WITH_PROVIDER // TODO: deprecate
-  EDITED // TODO: deprecate
-  ANNUAL_REVIEW // TODO: deprecate
-  REVIEWED // TODO: deprecate
-  UNPUBLISHED // TODO: deprecate
-  PUBLISHED // TODO: deprecate
-  PINNED // TODO: deprecate
-  UNPINNED // TODO: deprecate
```

Comment:
I think we'll have to run a migration.. can you double check if postgres/prisma can handle this? 

essentially you can't just delete enum types, especially if there are entries in the db already using these types

Reply:
Ah yeah that's a good point. So should I keep these in but also run a migration?

From testing things locally Prisma is okay with this, I guess the issue is the old entries in the DB that need to be removed.

Reply:
I think we will actually have to 
1. Migrate all the list item reminder audits to list item event,
2. migrate to delete the old audit reminders

seeing as we're changing logic (searching on event table rather than audit table) - but but there are currently annual reviews going on, so the new queries won't work for them

Alternatively.. we leave anything to do with reminder and audit/events alone and leave it for another ticket.

Reply:
Yeah I guess I didn't consider this in the scope of work when estimating the ticket.

In my head they seem like fairly complex migrations. Having to change an entry in one table to match the requirement of another table, especially with json data. 

Happy to do it in this ticket if you think it's easy, but I'd rather do it in a new 2 story points ticket.